### PR TITLE
Update fawry nas source request and response specs

### PR DESCRIPTION
**Pull request by guven-kurt-cko**
[Ticket](https://checkout.atlassian.net/browse/APM-3662)
- Update source > request and response objects for fawry

### CODE REVIEW

**Comment by ben-ahmady-cko** [comment]
Thanks, the GW team needs to review UPAPI PRs so I've forwarded to @Joshua-Bedeau-cko and Enrica Mameli to take a look at

**Review by ahmed-farghaly-cko — APPROVED** [review]

File: nas_spec/components/schemas/Payments/RequestSources/PaymentRequestFawrySource.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      description:
+        maxLength: 265
+        type: string
+        description: 'The payment description.'
+      customer_profile_id:
+        type: string
+        description: "The customer's id within merchant's system."
+      customer_email:
+        type: string
+        description: "The customer's email address."
```
Comment (Joshua-Bedeau-cko) [inline nas_spec/components/schemas/Payments/RequestSources/PaymentRequestFawrySource.yaml:24]: This is PII - We cannot release this until we have appropriate masking. 
https://checkout.atlassian.net/browse/GTW-7614

**Review by Joshua-Bedeau-cko — CHANGES_REQUESTED** [review]

File: nas_spec/components/schemas/Payments/RequestSources/PaymentRequestFawrySource.yaml (hunk 1 above)
Comment (saad-khan-cko) [reply nas_spec/components/schemas/Payments/RequestSources/PaymentRequestFawrySource.yaml:24]: @Joshua-Bedeau-cko I think this PR can be merged now as the task you mentioned was released to production

Comment (saad-khan-cko) [reply nas_spec/components/schemas/Payments/RequestSources/PaymentRequestFawrySource.yaml:24]: @Joshua-Bedeau-cko can you please approve the PR now ? As the comment was marked with the `request changes` state, I can't merge until you approve it

**Pull request merged**